concourse/steps/release.py

```diff
@@ -390,17 +407,49 @@ def validate(self):
     def apply(
         self,
     ):
+        # clean repository if required
+        worktree_dirty = bool(self.github_helper._changed_file_paths())
```

Comment:
_changed_file_paths is also a symbol only available from `git_helper` (not github_helper)